README.md

```diff
@@ -5,31 +5,45 @@ PyPI.
 
 
 ## Usage
-To use the action simply add the following lines in the end of your
-`.github/main.workflow`.
-
-```hcl
-action "Upload Python dist to PyPI" {
-  uses = "re-actors/pypi-action@master"
-  env = {
-    TWINE_USERNAME = "f'{your_project}-bot'"
-  }
-  secrets = ["TWINE_PASSWORD"]
-}
+
+To use the action add the following step to your workflow file (e.g.:
+`.github/workflows/main.yml`)
+
+
+```yml
+- name: Publish package
+  uses: pypi/gh-action-pypi-publish@master
+  with:
+    user: __token__
+    password: ${{ secrets.pypi }}
```

Comment:
I'd go for
```suggestion
    password: ${{ secrets.pypi_password }}
```
as it reads more intuitively...

Reply:
```suggestion
    password: ${{ secrets.pypi_token }}
```